### DISCUSSION TITLE
Bump only the minor or patch version of a dependency via a predefined package Manager

### DISCUSSION THREAD

**Discussion by kitos9112** [renovatebot/renovate#10123] [Request Help] [answered]
Hello,

Couldn't find much documentation around my use case so throwing a question in here to see if someone can answer it.

I'm using the `terraform-version` and `terragrunt-version` package managers and I maintain different files in my Git repository that point to different minor versions (e.g. 1x `folderA/.terraform-version` with 0.13.4 and 1x `folderB/.terraform-version` on 0.14.5). I'd like the `renovatebot` to only bump the latest patch version automatically, but do not alter the pinned minor version.  It may be straightforward, but ideally, I'd not like to hardcode major/minor versions in my renovate `packageRules` section.

Thanks!

**Comment by rarkins** [comment, marked as answer]
```
{
  "packageRules": [{
    "description": "Split patch and minor updates",
    "matchManagers": ["terraform-version", "terragrunt-version"],
    "separateMinorPatch": true
  }, {
    "description": "Disable major and minor updates",
    "matchManagers": ["terraform-version", "terragrunt-version"],
    "matchUpdateTypes": ["major", "minor"],
    "enabled": false
  }]
}
```

**Comment by kitos9112** [comment]
That's it! I was missing the second element in the array!
